From: Reinhard Tartler <siretart@tauware.de>
Description: Revert upstream commit 5459f097a

This is to avoid an unlreased copy opencontainers/image-spec

Index: golang-github-containers-common/libimage/manifests/manifests.go
===================================================================
--- golang-github-containers-common.orig/libimage/manifests/manifests.go
+++ golang-github-containers-common/libimage/manifests/manifests.go
@@ -375,12 +375,9 @@ func (l *list) Add(ctx context.Context,
 			}
 			if instanceInfo.OS == "" {
 				instanceInfo.OS = config.OS
-				instanceInfo.OSVersion = config.OSVersion
-				instanceInfo.OSFeatures = config.OSFeatures
 			}
 			if instanceInfo.Architecture == "" {
 				instanceInfo.Architecture = config.Architecture
-				instanceInfo.Variant = config.Variant
 			}
 		}
 		manifestBytes, manifestType, err := src.GetManifest(ctx, instanceInfo.instanceDigest)
